internal/jobs/launch_instance_aws.go

```diff
@@ -87,13 +90,11 @@ func handleLaunchInstanceAWS(ctx context.Context, args *LaunchInstanceAWSTaskArg
 	}
 
 	logger.Trace().Msg("Executing RunInstances")
-	instances, awsReservationId, err := ec2Client.RunInstances(ctx, args.Detail.Name, args.Detail.Amount, types.InstanceType(args.Detail.InstanceType), args.AMI, pk.Name, userData)
+	instances, awsReservationId, err := ec2Client.RunInstances(ctx, args.Detail.Name, args.Detail.Amount, types.InstanceType(args.Detail.InstanceType), args.AMI, reservation.Detail.PubkeyNameAWS, userData)
```

Comment:
so the Name should be now saved to PubkeyNameAWS and passed to this method from there.
It feels like workaround and I belive it would be great if we have job outputs, but Happy to discuss later and use workaround for now.

Reply:
Not sure what you mean, this looks okay to me, like a stable and maintainable code. I don't think we are getting job pipeline anytime soon, even dbjobqueue does not support it (it is stored in DB yes but dependant jobs cannot access it I think).

Reply:
right :) well workaround because we fetch record from database just to get output from previsous job (and the "column" exists just to be passed from one job to the other ;))